src/lib/bot.ts

```diff
@@ -50,48 +100,102 @@ client.on('messageReactionAdd', async (reaction) => {
     const timeInSeconds =
       (Date.now() - reaction.message.createdTimestamp) / 1000;
     console.log(timeInSeconds);
-    if (timeInSeconds > 30) return reaction.message.react(delayed_react);
+    if (timeInSeconds > 30) {
+      reaction.message.react(delayed_react);
+      return;
```

Comment:
Here it's fine to return void from the function, we just want to stop current function's the execution